extensions/expanded-opcodes/README.md

```diff
@@ -0,0 +1,87 @@
+# General design
+
+**Extension State: Under Development**  
+**Enabled by Default: Yes**  
+**Requires: Base, VWI**  
+**CPUID Bit: 9**
+
+# Overview
+
+This extension adds an expanded instruction format to allow for a larger number of opcodes to be used. It also includes a relative jump and call with larger displacements.
+
+# Added Instructions
+
+The following opcodes are now defined.
+
+| First byte    | Second Byte  | Third Byte   | Comment                                                                   |
+|:--------------|:-------------|:-------------|:--------------------------------------------------------------------------|
+| `111 00 000`  | `SS CCCCCC`  | `RRR RRR MM` | Register-Register operation                                               |
+| `111 00 ???`  | `SS CCCCCC`  | `RRR RRR MM` | When `???` is not `000`, reserved                                         |
+| `111 01 000`  | `SS CCCCCC`  | `RRR IIIII`  | Register-Immediate operation                                              |
+| `111 01 ???`  | `SS CCCCCC`  | `RRR IIIII`  | When `???` is not `000`, reserved                                         |
```

Comment:
For consistency, (ease of decoding, etc.) why not use `111 CCCCC FF SS CCCC`? this makes the location of the FF and SS bits match within the byte, which should reduce the number of multiplexers needed to build efficient decoders.

This part is obviously very in flux anyway, I expect it to change a few times.

Reply:
With the shifting of the opcodes so that the base opcodes fit here as well, the least significant 4 bits being `CCCC` doesn't really fit unless you're specifically just moving the FF and SS bits so that they match.

Reply:
But there's no real reason for the base opcodes to have to fit here. That's just tossing away 16 opcodes for no advantage (they have to be decoded separately anyway!) And yes, the suggestion is to move those bits so that they match.

Reply:
I agree with Grape here, I prefer `FF SS CCCC` (We will see about `CCCCC` in the first byte when other extensions come I think). I also don't see a reason for the base opcodes to fit here.